flink-core/src/main/java/org/apache/flink/configuration/AkkaOptions.java

```diff
@@ -69,9 +71,10 @@ public static boolean isForceRpcInvocationSerializationEnabled(Configuration con
 
     /** Timeout for akka ask calls. */
```

Comment:
```suggestion
    /** Timeout for pekko ask calls. */
```
It might be reasonable to modify the JavaDoc here (and the other AkkaOptions entries) as well. ...since we're touching JavaDoc in other cases as well.